c31-pryhaven/nutshell2
# BM-tasks-5 Completed task feature

# Description

Added completed task checkbox feature and ERD.pdf

Applies to issue # (issue number)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Please checkout branch bm-tasks-5 and test the complete tasks checkbox feature. When box is checked, isComplete key:value should be changed from false to true, the DOM should update and the task should disappear, however, the task should still be present with isComplete = true in the json.

# Checklist:
